test/mitmproxy/test_proxy.py

```diff
@@ -42,11 +42,12 @@ def test_simple(self):
         assert self.p()
 
     def test_certs(self, tdata):
-        self.assert_noerr(
-            "--cert",
-            tdata.path("mitmproxy/data/testkey.pem"))
-        with pytest.raises(Exception, match="does not exist"):
-            self.p("--cert", "nonexistent")
+        with pytest.raises(Exception, match="ambiguous option"):
+            self.assert_noerr(
+                "--cert",
+                tdata.path("mitmproxy/data/testkey.pem"))
+            with pytest.raises(Exception, match="does not exist"):
+                self.p("--cert", "nonexistent")
```

Comment:
Is this change intentional? The double-indented `.raises` doesn't make much sense to me.

Reply:
Indeed double .raises doesn't make sense, removed the original, since the exception is now different, "ambiguous option" --certs and --cert-passphrase resembles each others and it now complains about that.